R/univariate_pipeline.R

```diff
@@ -93,7 +93,7 @@ univariate_analysis_pipeline <- function(
   # Filter X based on missingness, MAF, and variance
   if (!is.null(imiss_cutoff) || !is.null(maf_cutoff)) {
     # FIXME: I think we should use filter_X directly for univariate without considering Y? 
-    X_filtered <- filter_X(X, imiss_cutoff, maf_cutoff, var_thresh = xvar_cutoff, X_variance = X_variance)
+    X_filtered <- filter_X(X, imiss_cutoff, maf_cutoff, var_thresh = xvar_cutoff, maf = maf, X_variance = X_variance)
```

Comment:
We need it here only because x_scalar is unnamed.

Reply:
do we need any additional changes at this ?